finagle-thrift/src/main/scala/com/twitter/finagle/thrift/RichRequestHeader.scala

```diff
@@ -39,6 +39,7 @@ private[finagle] class RichRequestHeader(val header: RequestHeader) extends AnyV
       if (header.isSetParent_span_id) Some(SpanId(header.getParent_span_id)) else None,
       SpanId(header.getSpan_id),
       if (header.isSetSampled) Some(header.isSampled) else None,
-      if (header.isSetFlags) Flags(header.getFlags) else Flags()
+      if (header.isSetFlags) Flags(header.getFlags) else Flags(),
+      None // TODO: Regenerate tracing.thrift, and update to support high bits in 128 TraceID
```

Comment:
Can we do this as part of this PR? Outside of Twitter, thrift is the most heavily used RPC protocol, so it would be a shame to not have it.

Reply:
@bryce-anderson I commented on this in the discussion part of the PR, but it's probably gotten buried. I tried using thrift 0.9 and 0.10 (both installed via homebrew) to regenerate tracing.thrift using the commands at the top of the file, and these resulted in code that wouldn't compile. There were numerous references to apache libs which weren't resolved.  Do you know if there have been manual modifications to the generated files, or if you all are using a modified version of thrift?

Reply:
We use old thrift, libthrift-0.5. ☹️